tests/ui/perfherder/alerts-view/alerts_test.jsx

```diff
@@ -869,6 +917,23 @@ test('Sherlock status 0 in tooltip on alerts', async () => {
   await waitFor(() => getByText(alertBackfillResultVisual.preliminary.message));
 });
 
+test(`Sherlock status 0 in tooltip on alerts Debug Tools column not contains side-by-side icon`, async () => {
```

Comment:
What is the link between Sherlock statuses and the Debug Tools column?
Why do we need to check the Debug Tools column based on the Sherlock statuses?

Reply:
The side-by-side job is triggered right before the backfill are triggered, within [BackfillTool.backfill_job](https://github.com/mozilla/treeherder/blob/master/treeherder/perf/auto_perf_sheriffing/backfill_tool.py#L28). Sherlock status from 2 above means that the code got past the line where side-by-side is triggered, thus it was triggered.
Side-by-side usually finishes in at most 30 minutes from when it was triggered.

Reply:
We should make these test names more readable.
Maybe we could also add the name for Sherlock statuses and then have something like this:

```suggestion
test(`Side-by-side icon is not displayed in Debug Tools column when Sherlock status is 0 (Not backfilled) in tooltip alerts`, async () => {
```

Let me know what you think.